starter_scripts/exaslct_within_docker_container_slim.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+#set -e => immediately exit if any command [1] has a non-zero exit status
+#set -u => reference to any variable you haven't previously defined is an error and causes the program to immediately exit.
+#set -o pipefailt => This setting prevents errors in a pipeline from being masked.
+#                    If any command in a pipeline fails,
+#                    that return code will be used as the return code of the whole pipeline.
+set -euo pipefail
+
+
+RUNNER_IMAGE_NAME="$1"
+shift 1
+
+if [[ -t 1 ]]; then
+  terminal_parameter=-it
+else
+  terminal_parameter=""
+fi
+
+RED='\033[0;31m'
+printf "${RED} Using slim version of exaslct. Please note that all input/output artifacts (input flavor directories, output, log, etc.) needs to be within the current working directory.\n"
+
+
+
+[[ "$(uname)" = Darwin ]] && rl=greadlink || rl=readlink
```

Comment:
Not sure, if I like this construct. It is shorter than an if, but I feel it has more potential for errors.